doc/sphinx-guides/source/api/auth.rst

```diff
@@ -0,0 +1,63 @@
+API Tokens and Authentication 
+=============================
+
+An API token is similar to a password and allows you to authenticate to Dataverse APIs to perform actions as you. Many Dataverse APIs require the use of an API token.
+
+.. contents:: |toctitle|
+    :local:
+
+How to Get an API Token
+-----------------------
+
+Your API token is unique to the server you are using. You cannot use your API token from one server on another server.
```

Comment:
More better: "_You cannot take your API token from one server and use it on another server._"